01_01_Basic_Rules.txt

```diff
@@ -425,6 +421,19 @@ class has a Proficiency with a given armor type then they can purchase or wear
 that armor type in combat. If your character doesn't have that proficiency, then
 they can't.
 
+===== Assigning Skills to a Character =====
+
+	Characters gain points in a wide variety of skills to represent their experience
+and expertise in different fields and activities. Every time your character levels up,
+they gain skill points equal to double your Skill Point Gain. Assign each skill point to
+a skill as you see fit; see CharacterCreation/06_Skills for details about skills and for
+a list of commonly-used skills in Compound X. At level 1, your character will have two
+times their Skill Point Gain to assign at will. You may gain a maximum of 60 skill

```

Comment:
I am concerned that we may be sending mixed signals by putting "Assign ... as you see fit" and "to assign at will" before the restrictions of the 60 point cap and 6 points per level cap. Nonblocking.

Reply:
Yeah, I was iffy on that while writing it